trappy/base.py

```diff
@@ -179,6 +179,12 @@ def append_data(self, time, comm, pid, cpu, line, data):
         self.line_array.append(line)
         self.data_array.append(data)
 
+    def conv_to_int(self, value):
```

Comment:
Maybe we can use a less ugly name as well as make this function a bit more generic just by changing its name to something like `type_cast(self, value)`.

Than it would be easier for a future patch to add in additional type casting (e.g. floats, hex)... potentially we can also map-translate known string tokens to values (e.g. for sched_switch) status events...